private/bufpkg/bufcheck/buflint/testdata/lint_stable_package_import/api/v1/foo.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package api.v1;
+
+import "api/v1alpha/bar.proto";
```

Comment:
An import on a stable package, and a package with no stability level, should also be added (both should not result in failures)